code/core/src/main/java/com/adobe/marketing/mobile/util/EventUtils.kt

```diff
@@ -22,28 +22,28 @@ private const val KEY_DEBUG_EVENT_TYPE = "eventType"
 private const val KEY_DEBUG_EVENT_SOURCE = "eventSource"
 
 /**
- * Returns the debug event type (identified by debug.eventType) from the event data if present, otherwise null.
- * @return the debug event type if present, otherwise null
+ * The debug event type (identified by debug.eventType) from the event data if present, otherwise null
  */
-fun Event.getDebugEventType(): String? {
-    val debugData = getDebugEventData() ?: return null
-    val debugEventType = debugData[KEY_DEBUG_EVENT_TYPE]
-    if (debugEventType !is String) return null
+val Event.debugEventType: String?
```

Comment:
Can you also change debugEventData to a property and simplify the implementation?
val Event.debugEventType: String?
    get() = debugEventData?.get(KEY_DEBUG_EVENT_TYPE) as? String

val Event.debugEventSource: String?
    get() = debugEventData?.get(KEY_DEBUG_EVENT_TYPE) as? String